rpd_generator/bdl_structure/bdl_commands/system.py

```diff
@@ -1330,6 +1394,73 @@ def populate_air_energy_recovery(self):
                 self.air_energy_recovery_outdoor_airflow
             )
 
+    def populate_fan_operation_during_occupied(self):
+        fan_sch_name = self.keyword_value_pairs.get(BDL_SystemKeywords.FAN_SCHEDULE)
+
+        if (
+            self.keyword_value_pairs.get(BDL_SystemKeywords.TYPE)
+            == BDL_SystemTypes.RESVVT
+        ):
+            return FanSystemOperationOptions.CYCLING
+
+        elif (
+            self.keyword_value_pairs.get(BDL_SystemKeywords.TYPE)
+            == BDL_SystemTypes.DOAS
+        ):
+            fan_sch = self.rmd.bdl_obj_instances.get(fan_sch_name)
+            if not fan_sch:
+                return FanSystemOperationOptions.INTERMITTENT
+            else:
+                return FanSystemOperationOptions.CONTINUOUS
```

Comment:
Similar to my comment above, what if the schedule has all 0 values then it would not be continuous " If the value is 0 or -1, the DOAS fans are off; regardless of the fan settings of the served systems. If a served system is operating when the DOAS fans are off, then no outdoor air is delivered." Maybe I am missing something that will become clearer as I go along.

Reply:
It seems like this comment is touching on "what is considered occupied" and I thought we were saying that occupied is just when the fan schedule is 1

Reply:
> It seems like this comment is touching on "what is considered occupied" and I thought we were saying that occupied is just when the fan schedule is 1

Hmmm, maybe the heating and coolign system should dictate this since the DOAS is decoupled.